### ISSUE TITLE
What is the best way to handle long audio files?

### ISSUE THREAD

**Issue by DizzyProtos** [mpariente/asteroid#203]
Hello! I need to separate speakers in long audio files (2min and more).
Could this be done with Asteroid? If yes, then what is the best way to achieve this?

I tried to do it like this, but it throws CUDA out of memmory error.
```
from asteroid.models import ConvTasNet

nnet = ConvTasNet.from_pretrained('SeparatorModels/ConvTasNet_LibriMix_sepclean.pth')
nnet.cuda(3)
nnet.separate('2_min_long.wav')
```
Here's the error
`RuntimeError: CUDA out of memory. Tried to allocate 14.68 GiB (GPU 3; 10.92 GiB total capacity; 527.54 MiB already allocated; 9.92 GiB free; 530.00 MiB reserved in total by PyTorch)`

**Comment by mpariente** [comment]
#191 has just been merged, you would need to use the LambdaOverlappAdd class, pass ConvTasNet as the lambda, choose the window size etc.. And it would work. 

We still need to add examples for it though. Would you like to add a code example on the class? That would be great ! 

Other refs #140

**Comment by DizzyProtos** [comment]
Thank you! I'll try that and add an example if it going to work out.

**Comment by DizzyProtos** [comment]
@mpariente Hello! I've been trying to use LambdaOverlappAdd class, but it takes about 15min for ConvTasNet model to process 2min long file using GPU. It seems like a very long time, is there some error in the LambdaOverlappAdd class or am I just doing it wrong?

Here's my code:
```
from asteroid.models import ConvTasNet
from asteroid.dsp.overlap_add import LambdaOverlapAdd
import soundfile as sf
import torch
import time


nnet = ConvTasNet(n_src=2)
nnet.cuda(3)
continuous_nnet = LambdaOverlapAdd(nnet=nnet,
                                   n_src=2, 
                                   window_size=128,
                                   window=None)
filename = <audio_path>
wav, fs = sf.read(filename, dtype="float32")
wav = torch.from_numpy(wav).to(device=torch.device('cuda:3'))
# Reshape 1dim to 3dim
wav = wav.unsqueeze(0).unsqueeze(1)

start_time = time.time()
out_wavs = continuous_nnet.forward(wav)
print("--- %s seconds ---" % (time.time() - start_time))
```

**Comment by mpariente** [comment]
The window size seems too small, 128 samples is almost nothing.
Try with something like 64000 (8s), it should work better.

**Comment by mpariente** [comment]
Solved in #221
